tfe/resource_tfe_organization_token.go

```diff
@@ -0,0 +1,107 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFEOrganizationToken() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEOrganizationTokenCreate,
+		Read:   resourceTFEOrganizationTokenRead,
+		Delete: resourceTFEOrganizationTokenDelete,
+
+		Schema: map[string]*schema.Schema{
+			"organization": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"force_regenerate": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"token": &schema.Schema{
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+		},
+	}
+}
+
+func resourceTFEOrganizationTokenCreate(d *schema.ResourceData, meta interface{}) error {
+	tfeClient := meta.(*tfe.Client)
+
+	// Get the organization name.
+	organization := d.Get("organization").(string)
+
+	log.Printf("[DEBUG] Check if a token already exists for organization: %s", organization)
+	_, err := tfeClient.OrganizationTokens.Read(ctx, organization)
+	if err != nil && err != tfe.ErrResourceNotFound {
+		return fmt.Errorf("Error checking if a token exists for organization %s: %v", organization, err)
+	}
+
+	// If error is nil, the token already exists.
+	if err == nil {
+		if !d.Get("force_regenerate").(bool) {
+			return fmt.Errorf("A token already exists for organization: %s", organization)
+		}
+		log.Printf("[DEBUG] Regenerating existsing token for organization: %s", organization)
```

Comment:
_minor_ `existsing` -> `existing`

Reply:
Fixed... Both in there and in the `team_token` resource...